src/test/java/br/com/caelum/comidinhas/tipoCozinha/EditarTipoCozinhaValidatorTest.java

```diff
@@ -0,0 +1,40 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.junit.jupiter.api.*;
+import org.springframework.validation.Errors;
+
+import static org.mockito.Mockito.*;
+
+class EditarTipoCozinhaValidatorTest {
+
+    private TipoCozinhaRepository tipoCozinhaRepository;
+    private Errors errors;
+    private TipoCozinhaInputEditar tipoCozinhaInputEditar;
+    private EditarTipoCozinhaValidator editarTipoCozinhaValidator;
+
+    @BeforeEach
+    public void setup(){
```

Comment:
O `setup` não precisa ser público.

```suggestion
    void setup(){
```